### PR TITLE
feat(bulk-ui): Create user interface for bulk operations

### PR DESCRIPTION
**Pull request by dezse97**
The PR contains:
- create new bulk
- edit a bulk
- view the history configuration changes
- activate changes
- order bulks
- delete bulk
- duplicate bulk
- created a common reusable component for the points table

UI tests coming soon.

Please suggest design improvements:
Screenshot 2021-11-05 at 16 15 42
Screenshot 2021-11-05 at 16 15 36
Screenshot 2021-11-05 at 16 15 25

### CODE REVIEW

**Comment by burgerni10** [comment]
@jfhenon I wonder if the bulk menu should be accessible from the engine menu on the main page, under the 'Add South' item ?

@dezse97 : you can remove the breadcrumb on the Activation and Bulk pages.
On specific bulk, you can take the same sub-nav-bar than the south points configuration page with a "back" button. You can add the ongoing status inside the sub nav bar.

**Comment by dezse97** [comment]
> @jfhenon I wonder if the bulk menu should be accessible from the engine menu on the main page, under the 'Add South' item ?
> 
> @dezse97 : you can remove the breadcrumb on the Activation and Bulk pages. On specific bulk, you can take the same sub-nav-bar than the south points configuration page with a "back" button. You can add the ongoing status inside the sub nav bar.

@burgerni10 the second part of your comment is done. Updated the screenshots. The first part is doable as well, only decide how do you want :)

**Comment by jfhenon** [comment]
> > @jfhenon I wonder if the bulk menu should be accessible from the engine menu on the main page, under the 'Add South' item ?

I would rather add it to the top toolbar.

File: src/client/services/apis.js [hunk 1]
```diff
@@ -42,6 +42,12 @@ const getConfig = () => getRequest('/config')
 const updateConfig = (body) => putRequest('/config', body)
 const activateConfig = () => putRequest('/config/activate')
 
+const getHistoryConfig = () => getRequest('/historyQuery/config')
+const updateHistoryConfig = (body) => putRequest('/historyQuery/config', body)
+const activateHistoryConfig = () => putRequest('/historyQuery/config/activate')
```
Comment (burgerni10) [inline src/client/services/apis.js:47]: url path should be history-query instead of historyQuery

File: src/client/index.jsx [hunk 2]
```diff
@@ -19,19 +20,22 @@ import About from './About/About.jsx'
...
+      <ConfigProviders>
         <AlertProvider>
           <TopHeader />
           <Container className="oi-container-with-top-nav" fluid>
             <AlertContainer />
             <Routes>
               <Route exact path="/" element={<HomePage />} />
               <Route exact path="/engine" element={<Engine />} />
+              <Route exact path="/historyQuery" element={<HistoryQuery />} />
+              <Route exact path="/historyQuery/:id" element={<ConfigureHistoryQuery />} />
```
Comment (burgerni10) [inline src/client/index.jsx:38]: url path should be history-query instead of historyQuery

Comment (burgerni10) [reply src/client/index.jsx:38]: To be adapted in the navigate call too

File: src/client/style/oi.css [hunk 3]
```diff
@@ -154,9 +154,15 @@ a:focus {
...
 .oi-add-button {
-float: right;
-margin-right: 10px;
-margin-top: -7px;
+  float: right;
+  margin-right: 10px;
+  margin-top: -7px;
+}
+
+.oi-add-bulk {
```
Comment (burgerni10) [inline src/client/style/oi.css:162]: oi-add-history-query

**Review by burgerni10 — CHANGES_REQUESTED** [review]
Small changes, otherwise ok for me

**Comment by dezse97** [comment]
Changes are done @burgerni10. Can be merged from my side. Will continue on another branch

**Review by burgerni10 — APPROVED** [review]

**Pull request merged**